pubspec.lock

```diff
@@ -384,6 +384,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "4.1.0"
+  lint:
```

Comment:
we'd better ignore this file!

Reply:
In applications, I think it's recommended to add this file, since it contains a locked version for each package.
https://dart.dev/guides/libraries/private-files